ideascube/utils.py

```diff
@@ -47,3 +47,8 @@ def to_unicode(text):
         except UnicodeDecodeError:
             text = text.decode('latin-1')
         return text
+
+
+def iter_file(file):
+    for row in file:
+        yield to_unicode(row)
```

Comment:
Oh no, you did it.

Sorry, I should have said earlier that this wouldn't work, see [my previous comment](https://github.com/ideascube/ideascube/pull/610#discussion_r86303725)

Please move back to what you had before.